### DISCUSSION TITLE
I can't get rid of Zsh

### DISCUSSION THREAD

**Discussion by EmilianoM223** [ohmyzsh/ohmyzsh#11594] [Help] [answered]
I installed it and quickly realized it wasn't for me, so I followed the instructions, but even after I restarted the terminal it doesn't let me get rid of it. Any help is really appreciated.

**Comment by carlosala** [comment, marked as answer]
If you don't want to use zsh, you can use [chsh](https://linux.die.net/man/1/chsh) to change your shell.
